build.gradle

```diff
@@ -552,9 +552,27 @@ configure(dexerProjects) { subproject ->
                 mkdir "${subproject.buildDir}/kotlin/d8"
             }
 
+            //accessing subproject.configurations.implementation.asPath failed due to it not being configurable as resolvable
+            // following this thread I came up with the solution below to get the paths of each jar dependencies
+            // https://discuss.gradle.org/t/converting-maven-plugin-to-gradle-plugin/35863/9
+            def classPaths = []
+            subproject.configurations.each { conf ->
+                if (conf.name.endsWith("implementation")) {
+                    def copiedConfiguration = conf.copyRecursive();
+                    copiedConfiguration.setCanBeResolved(true);
+                    copiedConfiguration.asPath.split(';').each { path ->
+                        classPaths += ['--classpath', path]
+                    }
+                }
+            }
+            // classPaths += ['--classpath', "c:/Program Files (x86)/Java/jre1.8.0_241/lib/rt.jar"]
```

Comment:
I think there is a better solution, I remember having the same issue with atrium-bc-tests and the solution was to define an own configuration. You could do something like:
```
configure(dexerProjects) { subproject ->
    configurations {
        d8Classpath
    }
    dependencies {
        d8Classpath subproject
    }
```
and then somehting along the line of
```
def classPaths = subproject.configurations.d8Classpath.files.collect {
    ["--classpath", it.getAbsolutePath()]
}.flatten()
```